tests/Fixtures/e2e/Test_PHP_Options_Config/infection.json

```diff
@@ -0,0 +1,13 @@
+{
+    "timeout": 25,
+    "source": {
+        "directories": [
+            "src"
+        ]
+    },
+    "logs": {
+        "summary": "infection.log"
+    },
+    "tmpDir": ".",
+    "initialTestsPhpOptions": "-d short_open_tag=Test"
```

Comment:
@maks-rafalko I used short_open_tag ini settings, but this will probably cause problems with php7.4/8. Any suggestions on another one to use?

Thanks

Reply:
would something like `-d memory_limit=123M` work for you?